fractal_server/app/runner/_slurm/__init__.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from ...models import Workflow
+from .._common import recursive_task_submission
+from ..common import async_wrap
+from ..common import TaskParameters
+from .executor import FractalSlurmExecutor
+
+
+"""
+Slurm Bakend
+
+This backend runs fractal workflows as separate processes using a python
+thread process pool, where each thread is responsible for running a single
+task in a subprocess.
+
+Incidentally, it represents the reference implementation for a backend.
+"""
+
+
+def _process_workflow(
+    *,
+    workflow: Workflow,
+    input_paths: List[Path],
+    output_path: Path,
+    input_metadata: Dict[str, Any],
+    logger_name: str,
+    workflow_dir: Path,
+    username: str = None,
+    worker_init: Optional[
```

Comment:
The same comment in the process runner was meant to be here. Possibly we'll have to look at this variable again, but let's think about it when appropriate.

(nothing to do here)